assets/panel/debugger.properties

```diff
@@ -707,7 +707,7 @@ functionSearchSeparatorLabel=←
 # Do not localize "CmdOrCtrl+;", or change the format of the string. These are
 # key identifiers, not messages displayed to the user.
 gotoLineModal.placeholder=Go to line…
-gotoLineModal.key=CmdOrCtrl+;
+gotoLineModal.key2=CmdOrCtrl+;
```

Comment:
Can we split the part of comment about the shortcut into a separate comment? Right now it's on the placeholder string's comment.